packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -46,70 +54,57 @@ The [Brands API](/docs/reference/api/brands/) is a feature (currently in Early A
 
 - If you customize an email template, you need to manually create a translation for each additional language that you support in your org.
 
-## Add or edit email templates
+## Edit a default email template
 
 Use these steps to add or edit a template in one of the Okta-supported languages.
 
 > **Note:** Free trial editions of Okta can't create or send customized email templates.
```

Comment:
"can't create or send customized email templates without contacting support@okta.com to request this feature"

Reply:
I'll add this note to the "What you need" section of the nutrition facts at the top of the page.